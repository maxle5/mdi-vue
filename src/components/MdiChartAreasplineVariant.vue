
  <template>
      <svg :width="size" :height="size" :viewBox="viewBox">
          <path d="M22,21H2V3H4V15.54L9.5,6L16,9.78L20.24,2.45L21.97,3.45L22,21Z" style="fill: currentColor" />
      </svg>
  </template>

  <script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiChartAreasplineVariant",
      props: {
          size: { 
              type: Number, 
              default: 24 
          }
      },
      computed: {
        viewBox() {
          return `0 0 ${this.size} ${this.size}`;
        }
      }
  })
  </script>
