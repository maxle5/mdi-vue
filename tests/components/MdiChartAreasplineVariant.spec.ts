
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiChartAreasplineVariant from "../../src/components/MdiChartAreasplineVariant.vue";

test("MdiChartAreasplineVariant snapshot", () => {
  const wrapper = mount(MdiChartAreasplineVariant, {});
  expect(wrapper.html()).toMatchSnapshot();
});
