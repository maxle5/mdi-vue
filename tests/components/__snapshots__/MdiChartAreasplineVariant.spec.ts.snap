// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiChartAreasplineVariant snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M22,21H2V3H4V15.54L9.5,6L16,9.78L20.24,2.45L21.97,3.45L22,21Z" style="fill: currentColor;"></path>
</svg>"
`;
